platform/platform.go

```diff
@@ -128,6 +128,74 @@ func (t *TestCluster) DropFile(localPath string) error {
 	return nil
 }
 
+// Copy a file between two machines in a cluster.
+func TransferFile(src Machine, srcPath string, dst Machine, dstPath string) error {
```

Comment:
i suppose you need a io.ReadCloser wrapper around `ssh cat` that cleans up ssh on `Close()`